pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,63 @@
+We have a "master" code branch, a "Master Sync" feature, an "EngineMaster" object, and more.
+We are working to migrate our code away from these words, even in cases where the particular meaning of "master" was never explicitly associated with a problematic context.
+
+It’s important to note that there will be some cases where the word will remain.
+Many DJ controllers have buttons marked "master" on them, and we need to refer to those buttons by name.
+There are also the audio engineering concepts of a "master tape" or "audio mastering", so there may be areas of our documentation or code that would be less understandable if we tried to use synonyms.
+But in situations where master is used in code and previously was part of a master/slave terminology pairing, we commit to changing the language we use by DATEDATEDATE **TKTKTKTK commit to a timeline to change the main branch?? TKTKTKTKTK**.
+
+But changing language is not enough.
+
+## Diversifying Recruiting
+
+In our discussions, we quickly noted that the Mixxx contributor base is largely white, male, and either from the US or Western Europe.
+For some of our coders, the experience of working on Mixxx has been a doorway to employment at some of the most prestigious software companies, and we want that door to be open to anyone.
+It is our goal to make the Mixxx project as inclusive, accessible and welcoming as possible.
+As such, it is our responsibility to actively seek out and promote the participation of traditionally underserved groups.
+
+As a small volunteer-led project with no regular income, budget, or employees, we don't yet know for sure what the demographic makeup of our contributors is -- but we are aware of the research that shows that blind admission and hiring tends to result in less diverse outcomes.
+To that end we plan to participate in the [Outreachy](https://www.outreachy.org/) program and have a paid internship position by December 2020, the next available cycle.
+Outreachy’s stated goal is to provide "internships to work in open source and free software, for women (both cis and trans), trans men, and genderqueer people" and "residents and nationals of the United States of any gender who are Black/African American, Hispanic/Latin@, Native American/American Indian, Alaska Native, Native Hawaiian, or Pacific Islander."
+They encourage "anyone who faces under-representation, systemic bias, or discrimination in the technology industry of their country" to apply.
+
+Our goal is that this Outreachy intern will be the first of an increasing number of valued contributors from underserved communities that will help us be a healthier, more sustainable open source project.
+With your help, we intend to continue the internship process semiannually for at least 5 cycles and at that time assess if we should continue with the program or if our diversification efforts would be better served with a new approach.
```

Comment:
```suggestion
With your help, we intend to continue participating in Outreachy as long as the community keeps funding it.
```
I don't understand why we would stop doing this?

Reply:
Let's try it first and evaluate.
Who knows, maybe there are / will be projects that are better in some regard. For example the same concept on a global scale.

Btw, that's also the reason I'd rather omit "for at least 5 cycles".

Reply:
We should figure out our specific intent on Zulip

Reply:
One of the asks I've seen from POC is that white people commit to specific and ongoing efforts to fight racism, which is why I think setting a number is important, although it could certainly be smaller than 5. Goal setting also increases accountability.

I added the opportunity to reevaluate since it's entirely possible a different program that is even more suited to the Mixxx community will be available in the future.

Reply:
"as long as the community keeps funding it" is specific and ongoing.

Reply:
but maybe we should set a fundraising goal for more than one round?